dags/example-dag.py

```diff
@@ -1,3 +1,4 @@
+# Add file to PR
```

Comment:
The thought behind leaving this, was that you would want something to report on, other wise it would just be a dag reporting on itself. if you don't think that is confusing I can remove the sample dag

Reply:
removed